packages/evolution-backend/src/services/adminExport/exportToCsvByObjectWorkerPool.ts

```diff
@@ -75,6 +94,14 @@ const getPaths = function (parentPath, _object, attributes: string[] = []) {
         if (attribute === '_actions') {
             continue;
         }
+        if (
+            Array.isArray(_object[attribute]) &&
```

Comment:
Same as the other stuff. Can you store that in a variable?

 Why do you need to check if it's an array and not on the other one? Isn't it supposed to be the same as the if statement on lines 55-56?

Reply:
Done, the if was extracted in a `isArrayOfObjects` function